index.css

```diff
@@ -0,0 +1,161 @@
+html,
+body {
+    margin: 0;
+    font-family: 'Karla', sans-serif;
+    background-color: #219ebc;
+}
+
+/*typography*/
+
+h1,
+h2,
+h3 {
+    text-align: center;
+}
+
+h1,
+h2 {
+    color: #EED6D3;
+    margin: 0;
+}
+
+h1 {
+    font-size: 28px;
+}
+
+h2 {
+    font-size: 16px;
+}
+
+h3 {
+    margin: 25px;
+}
+
+/* layout */
+
+header {
+    background-color: #67595E;
+}
+
+.header-inner {
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+    width: 370px;
```

Comment:
I wouldn't use an explicit width here, just use justify-content to center things horizontally.

If you want the text to be directly in the center, I'd set the cat image to be an explicit width (looks like you already are), then use margin-right on the text container to balance it out (or use a grid)

Reply:
Addressed. Removed explicit width callout and replaced with justify-content: center. 

Not sure how the second half of the comment is relevant, could use some further clarification.